packages/sui-cz/index.js

```diff
@@ -45,11 +61,17 @@ module.exports = {
             return [{name: 'META'}]
           }
 
-          if (typesWithOtherScopes.indexOf(answers.type) > -1) {
-            return scopes.concat(otherScopes)
-          }
-
-          return scopes
+          return Promise.all(scopes.map(pkg => (
```

Comment:
We can also use the trick of the staged files to make sui-lint to execute only on changed files.
It would be WAY faster. What do you think?

Reply:
I guess they can't be lint related integration issues, so 👍

Reply:
Solved ! Thanks!